recipe/conda_forge_ci_setup/feedstock_outputs.py

```diff
@@ -140,7 +141,13 @@ def main(feedstock_name):
     """Validate the feedstock outputs."""
 
     if is_conda_forge_output_validation_on():
-        distributions = [os.path.relpath(p, conda_build.config.croot) for p in built_distributions()]
+        allowed_dist_names, allowed_subdirs = get_built_distribution_names_and_subdirs()
```

Comment:
This is an expensive operation. Can we cache this on disk using some python library.

Reply:
Yes let me check into that.

Reply:
example: https://stackoverflow.com/a/23526717/4768820

Reply:
yes this one too: https://pypi.org/project/cache-to-disk/